css/style.css

```diff
@@ -1,4 +1,259 @@
-/* Put your CSS here */
-html, body {
-    margin: 20px;
-}
\ No newline at end of file
+html,
+body {
+    margin: 0;
+    padding: 0;
+}
+
+body {
+    font: 14px Tahoma, Arial, sans-serif;
+    line-height: 1.4em;
+    color: #333;
+    width: 100%;
+    margin: 0 auto;
+}
+
+.mainTitle {
+    text-align: center;
+    margin: 50px auto;
+    color: #aaa;
+}
+
+#TasksSection {
+    background: #fff;
+    background: rgba(255, 255, 255, 0.9);
+    margin: 20px auto;
+    border: 1px solid #ccc;
+    position: relative;
+    width: 95%;
+}
+
+#header {
+    padding: 5px;
+    height: auto;
+    background-color: #CFF3AB;
+}
+
+#new-Tasks,
+.edit {
+    position: relative;
+    /* margin-left: 45px; */
+    width: calc(100% - 60px);
+    font-size: 24px;
+    line-height: 1.4em;
+    outline: none;
+    padding: 6px;
+    border: 1px solid #ddd;
+    -webkit-box-sizing: border-box;
+    -moz-box-sizing: border-box;
+    -ms-box-sizing: border-box;
+    -o-box-sizing: border-box;
+    box-sizing: border-box;
+}
+
+#main {
+    position: relative;
+    z-index: 2;
+    border-top: 1px solid #adadad;
+}
+
+
+
+#chk-all {
+    text-align: center;
+    width: 40px;
+    height: 20px;
+    border: none;
+}
+
+#Tasks-list {
+    margin: 0;
+    padding: 0;
+    list-style: none;
+}
+
+    #Tasks-list li {
+        position: relative;
+        font-size: 18px;
+        border-bottom: 1px solid #ccc;
+        padding: 0 10px;
+    }
+
+        #Tasks-list li:last-child {
+            border-bottom: none;
+        }
+
+        #Tasks-list li.editing {
+            padding: 0;
+        }
+
+            #Tasks-list li.editing .edit {
+                display: block;
+                width: calc(100% - 40px);
+                margin: 5px auto;
+            }
+
+            #Tasks-list li.editing .view {
+                display: none;
+            }
+
+        #Tasks-list li .chk {
+            text-align: center;
+            width: 40px;
+            height: auto;
+            position: absolute;
+            top: 0;
+            bottom: 0;
+            margin: auto 0;
+            border: none;
+            height: 17px;
+        }
+
+
+        #Tasks-list li label {
+            -moz-word-break: keep-all;
+            -o-word-break: keep-all;
+            word-break: keep-all;
+            padding: 15px;
+            margin-left: 45px;
+            display: block;
+            /* line-height: 1.2; */
+            -webkit-transition: color 0.4s;
+            -moz-transition: color 0.4s;
+            -ms-transition: color 0.4s;
+            -o-transition: color 0.4s;
+            transition: color 0.4s;
+        }
+
+        #Tasks-list li.completed label {
+            color: #777;
+            text-decoration: line-through;
+        }
+
+.remove {
+    display: none;
+    position: absolute;
+    top: 0;
+    right: 10px;
+    bottom: 0;
+    width: 40px;
+    height: 40px;
+    margin: auto 0;
+    font-size: 22px;
+    color: #C32323;
+    ;
```

Comment:
oh! it's my mistake 


     On Monday, September 28, 2015 4:49 PM, Ruan Naude <notifications@github.com> wrote:
   

 In css/style.css:> +            color: #777;
> +            text-decoration: line-through;
> +        }
> +
> +.remove {
> +    display: none;
> +    position: absolute;
> +    top: 0;
> +    right: 10px;
> +    bottom: 0;
> +    width: 40px;
> +    height: 40px;
> +    margin: auto 0;
> +    font-size: 22px;
> +    color: #C32323;
> +    ;
hummmm—
Reply to this email directly or view it on GitHub.